targets/common

```diff
@@ -45,6 +58,10 @@ elif [ -z "$TARGETNOINSTALL" -o -n "$RESTOREHOSTBIN" ]; then
     done
     # Source the prerequisites
     for t in $REQUIRES; do
+	# If it is already provided change it to the target what provides it.
+        if grep -q "^$t=.*\$" "${PROVIDESFILE:-/dev/null}"; then
+            t="`awk -F= '/'$t'=/{print $2}' "${PROVIDESFILE:-/dev/null}"`"
```

Comment:
Add double-quotes around the `$t` part.  Not really necessary here, but good practice.